src/instruments/src/EFB/Performance/Widgets/LandingWidget.tsx

```diff
@@ -108,7 +108,7 @@ export const LandingWidget = () => {
         });
     };
 
-    const syncValues = async (): Promise<void> => {
+    const handleSyncValues = async (): Promise<void> => {
```

Comment:
Why does this return a promise?

Reply:
Because it is performing an asynchronous operation and it does not resolve to a value since there is nothing to be returned from what the body of the function is intended to do.

Reply:
> Why does this return a promise? It doesn't look to be handled.

An async function _always_ returns a promise.